recipes/pywinrm/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pywinrm" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "3030f700fbd6d06f715d4374c10b3586624bccca003b7075dd281c875705ac1b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - six
+    - xmltodict
+    - requests >=2.9.1
+    - requests_ntlm >=0.3.0
+
+test:
+  imports:
+    - winrm
+
+about:
+  home: https://github.com/diyan/pywinrm
+  license: MIT
+  license_family: MIT
```

Comment:
No license file included?

Reply:
Added issue upstream:

https://github.com/diyan/pywinrm/issues/183

Reply:
manually packaged the license file in latest commit.